keyboards/keychron/q2/rev_0113/config.h

```diff
@@ -30,4 +30,4 @@
 #define ENCODERS_PAD_B { B5 }
 
 /* Specifies the number of pulses the encoder registers between each detent */
-#define ENCODER_RESOLUTION 2
+#define ENCODER_RESOLUTION 4
```

Comment:
I am not sure if this is just my board, but it was double stepping

Reply:
seems to be consistent here: https://github.com/qmk/qmk_firmware/blob/master/keyboards/keychron/q2/rev_0111/config.h#L33

Reply:
You mean that on 2, it was registering twice?

Reply:
correct

Reply:
Ideally, this should be changed at the keymap level, rather than at keyboard level.

Reply:
I think it is just incorrect right now - as it is 4 on the other model - https://github.com/qmk/qmk_firmware/blob/master/keyboards/keychron/q2/rev_0111/config.h#L33

Reply:
Is there a way to change it on the keymap? I think I get an error about overriding a `#define`

Reply:
`#undef ENCODER_RESOLUTION`

Reply:
how can I convince you it is a misconfiguration in the board, rather than set it for just my keymap?

Reply:
did the encoder ship with the board?  or as an add-on?

Reply:
Yeah, it ships as a SKU